File: Bars.Gkh/Entities/Dicts/ContentRepairMkdWork.cs
namespace Bars.Gkh.Entities.Dicts
{
    /// <summary>
    /// Работы по содержанию и ремонту МКД
    /// </summary>
    public class ContentRepairMkdWork : BaseGkhEntity
    {
        /// <summary>
        /// Вид работы
        /// </summary>
        public virtual Work Work { get; set; }

        /// <summary>
        /// Код
        /// </summary>
        public virtual string Code { get; set; }

        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Описание
        /// </summary>
        public virtual string Description { get; set; }
    }
}

File: Bars.Gkh/libs/B4/model/Dict/ContentRepairMkdWork.js
Ext.define('B4.model.dict.ContentRepairMkdWork', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ContentRepairMkdWork'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Work' },
        { name: 'Name' },
        { name: 'Code' },
        { name: 'Description' }
    ]
});
